packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -86,46 +91,81 @@ abstract class OneSequenceGestureRecognizer extends GestureRecognizer {
     assert(_entries.isEmpty);
   }
 
+  /// Causes events related to the given pointer ID to be routed to this recognizer.
+  ///
+  /// The pointer events are delivered to [handleEvent].
+  ///
+  /// Use [stopTrackingPointer] to remove the route added by this function.
   void startTrackingPointer(int pointer) {
     GestureBinding.instance.pointerRouter.addRoute(pointer, handleEvent);
     _trackedPointers.add(pointer);
     assert(!_entries.containsValue(pointer));
     _entries[pointer] = GestureBinding.instance.gestureArena.add(pointer, this);
   }
 
+  /// Stops events related to the given pointer ID from being routed to this recognizer.
+  ///
+  /// If this function reduces the number of tracked pointers to zero, it will
+  /// call [didStopTrackingLastPointer] synchronously.
+  ///
+  /// Use [startTrackingPointer] to add the routes in the first place.
   void stopTrackingPointer(int pointer) {
     GestureBinding.instance.pointerRouter.removeRoute(pointer, handleEvent);
     _trackedPointers.remove(pointer);
     if (_trackedPointers.isEmpty)
       didStopTrackingLastPointer(pointer);
   }
 
+  /// Calls [stopTrackingPointer] if the pointer with the given ID is being tracked by this recognizer.
   void ensureNotTrackingPointer(int pointer) {
     if (_trackedPointers.contains(pointer))
       stopTrackingPointer(pointer);
   }
 
+  /// Stops tracking the pointer associated with the given event if the event is
+  /// a [PointerUpEvent] or a [PointerCancelEvent] event.
   void stopTrackingIfPointerNoLongerDown(PointerEvent event) {
     if (event is PointerUpEvent || event is PointerCancelEvent)
       stopTrackingPointer(event.pointer);
   }
-
 }
 
+/// The possible states of a [PrimaryPointerGestureRecognizer].
```

Comment:
it's confusing when looking at this enum that there's no state for "accepted". The enum description should discuss why this is.